### DISCUSSION TITLE
Standalone Binaries with Laravel Octane

### DISCUSSION THREAD

**Discussion by coleshirley** [php/frankenphp#513] [Q&A]
I have followed the documentation to create a standalone binary for my laravel project and it works perfectly. The same project also uses sail with the frankenphp octane binary for local development.

Is there a way to compile the binary to use Laravel octane? Is it already doing that? Is there actually a difference between a binary using octane vs not using octane? The documentation isn't very clear on these points

**Comment by fpolli** [comment]
I think the difference between using Octane and not using Octane is that Octane adjusts some of the ways Laravel works by default to accommodate the Worker mode paradigm of remaining in memory. I could be wrong about that, but that is the impression I came away from the Octane docs with.
